Gemfile

```diff
@@ -5,13 +5,18 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
-# Use Puma as the app server
-gem 'puma', '~> 5.0'
+gem 'haml',                 '~> 4.0'
```

Comment:
Is there any reason you've chosen haml over erb or another templating engine?

Reply:
Yes. Since I wasn't allowed to use Rails as a framework for this task and had to learn a bit of Sinatra, I thought to learn haml too 😄 . I used erb before in other projects.